jupyterhub_traefik_proxy/toml.py

```diff
@@ -94,6 +94,8 @@ def _clean_resources(self):
         try:
             if self.should_start:
                 os.remove(self.toml_static_config_file)
+                if self.traefik_auto_https:
+                    os.remove(self.traefik_acme_storage)
```

Comment:
It seems like _clean_resources is called along with .stop() which "Will be called during teardown if should_start is True.".

So, it seems then to me, that this configuration guarantees that everything in this ACME storage will reset on start/stop cycles. I think that this is where the acquired certificates will reside, then they should probably only be removed if we really want that specifically, but if we want to just restart, we probably want to reuse our previously acquired certificates.